server/controllers/loginUser.js

```diff
@@ -0,0 +1,39 @@
+const jwt = require('jsonwebtoken');
+const loginQuery = require('../database/queries/loginQuery');
+const { comparePassword } = require('../utils/hashPassword');
+const getHashPassword = require('../database/queries/getHashPassword');
+require('env2')('.env');
+
+const checkLogin = (req, res) => {
+  const { email, password } = req.body;
+  loginQuery(email)
+    .then((result) => {
+      if (result.rowCount) {
+        getHashPassword(email)
+          .then((hashed) => {
+            comparePassword(password, hashed.rows[0].pasword, (err, resu) => {
```

Comment:
`pasword` or `password`
spelling errors on code are fatal, try to use va code extension to check spelling for you.
check this one https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

Reply:
when I print hashed.rows[0] I found pasword not password
so I used it